webextensions/manifest.json

```diff
@@ -19,30 +19,30 @@
     "bookmarks"
   ],
   "icons": {
-    "16": "resources/16x16-light.svg",
-    "20": "resources/20x20-light.svg",
-    "24": "resources/24x24-light.svg",
-    "64": "resources/icon64.png",
-    "94": "resources/icon94.png"
+    "16": "resources/icon.svg",
+    "20": "resources/icon.svg",
+    "24": "resources/icon.svg",
+    "64": "resources/icon",
```

Comment:
I don't think the 64 and 94 lines should be changed and the files should not be removed as they are still necessary.